sonar-python-plugin/src/main/java/org/sonar/plugins/python/IpynbNotebookParser.java

```diff
@@ -30,7 +30,7 @@
 
 public class IpynbNotebookParser {
 
-  public static final String SONAR_PYTHON_NOTEBOOK_CELL_DELIMITER = "\n#SONAR_PYTHON_NOTEBOOK_CELL_DELIMITER\n";
+  public static final String SONAR_PYTHON_NOTEBOOK_CELL_DELIMITER = "#SONAR_PYTHON_NOTEBOOK_CELL_DELIMITER\n";
```

Comment:
Do we still need this `\n` even with the append on each line of the aggregated source.

Reply:
Yes, we need to append `\n` to the source because the `lines()` iterator removes them. But for the delimiter, we also need to have it.